docs/root/version_history/current.rst

```diff
@@ -22,12 +22,12 @@ Minor Behavior Changes
 * ext_authz: added requested server name in ext_authz network filter for auth review.
 * file: changed disk based files to truncate files which are not being appended to. This behavioral change can be temporarily reverted by setting runtime guard ``envoy.reloadable_features.append_or_truncate`` to false.
 * grpc: flip runtime guard ``envoy.reloadable_features.enable_grpc_async_client_cache`` to be default enabled. async grpc client created through getOrCreateRawAsyncClient will be cached by default.
-* health_checker: exposing `initial_metadata` to GrpcHealthCheck in a way similar to `request_headers_to_add` of HttpHealthCheck.
```

Comment:
This line 19's `accept-encoding` should be fixed also https://github.com/envoyproxy/envoy/pull/20505/files#diff-486ddf71f632f6537c18c0735baa4eb49e33ce2646092a74f2ba7f63a346ed09R19

Reply:
done, thanks